pages/jdl.md

```diff
@@ -1,1447 +0,0 @@
-   1. [Available field types and constraints](#types_and_constraints)
-   1. [Available entity options](#entity_options)
-1. [Troubleshooting](#troubleshooting)
-1. [Issues and bugs](#issues)
-
-***
-
-## <a name="sample"></a> JDL Sample
-
-The Oracle "Human Resources" sample application has been translated into JDL, and is available [here](https://github.com/jhipster/jdl-samples/blob/master/Oracle-Human-Resources-sample.jdl).
-The same application is loaded by default in [JDL-Studio](https://start.jhipster.tech/jdl-studio/) and [JHipster IDE](https://www.jhipster.tech/jhipster-ide/) as well.
-
-If you're looking for more samples, there is a repository for that right [here](https://github.com/jhipster/jdl-samples).
-
-## <a name="howtojdl"></a> How to use it
-
-###  <a name="importingjdl"></a> Importing a JDL file
-You can then use JDL files to generate entities:
-  - Simply create a file with the extension '.jh' or '.jdl',
-  - Declare your applications, deployments, entities and relationships or create and download the file with [JDL-Studio](https://start.jhipster.tech/jdl-studio/) or [JHipster IDE](https://www.jhipster.tech/jhipster-ide/),
-  - If you are creating only entities in then run `jhipster import-jdl my_file.jdl` in your JHipster application's root folder.
-  - If you are creating applications then just run `jhipster import-jdl my_file.jdl` in a folder.
-
-and *Voilà*, you are done!
-
-If you work in a team, perhaps you would like to have multiple files instead of one.
-We added this option so that you don't manually concatenate all the files into one,
-you just have to run.
-
-    jhipster import-jdl my_file1.jdl my_file2.jdl
-
-If you do not want to regenerate your entities while importing a JDL, you can use the `--json-only` flag to skip the
-entity creation part and create only the json files in `.jhipster` folder.
-
-    jhipster import-jdl ./my-jdl-file.jdl --json-only
-
-By default `import-jdl` regenerates only entities that have changed, if you want all your entities to be regenerated
-then pass in the `--force`  flag.
-Please note that this will overwrite all your local changes to the entity files
-
-    jhipster import-jdl ./my-jdl-file.jdl --force
-
-If you want to use it in your project, you can add do so by doing:
-  - NPM: `npm install jhipster-core --save`
-  - Yarn: `yarn add jhipster-core`
-
-to install it locally, and save it in your `package.json` file.
-
-###  <a name="exportingjdl"></a> Exporting to a JDL file
-
-If you already have entities in your application and wish to have a JDL file, fear not! You don't have
-to write it from scratch as there's a sub-generator that does that for you.
-
-Simply do `jhipster export-jdl <FILE_NAME>` in your app's root folder and you'll have all your entities,
-relationships and options exporting in a single JDL file.
-Note: you can also not provide a file name to the sub-generation, a default one will be chosen.
-
----
-
-## <a name="jdllanguage"></a> The language
```

Comment:
This could be useful for JDL developers, so may be we can have it the troubleshooting section?

Reply:
Good idea